scripts/hpp_genxr.py

```diff
@@ -149,7 +149,9 @@ def makeGenOpts(args):
             defaultExtensions='openxr',
             addExtensions=None,
             removeExtensions=None,
-            emitExtensions=emitExtensionsPat)
+            emitExtensions=emitExtensionsPat,
+            protectProto='#ifndef',
```

Comment:
ahh, does this actually affect whether or not the static dispatcher works? I bet it does...